sites/friday/src/manage-resources/resource-init.md

```diff
@@ -106,79 +172,223 @@ platform integration:add \
+environment:merge --resources-init=manual
+```
+
+<--->
+
++++
+title=In the Console
++++
+
+1. Navigate to the environment you want to merge into its parent.
+2. Click {{< icon merge >}} **Merge**.
+3. Select an initialization strategy from the proposed list.
+5. Click **Merge**.
+
+{{< /codetabs >}}
+
+### Environment activation
+
+When you activate an environment, {{% vendor/name %}} uses the same resource allocation as the parent environment.
+If there is no parent environment, the [`default` resource initialization strategy](#default-resources) applies.
+
+You can also set another resource initialization strategy using the CLI.
+
+{{% note theme="info" title="Prerequisite" %}}
+
+Make sure `sizing_api_enabled` is set to `true` in your `{{% vendor/configfile "app" %}}` file.
+
+{{% /note %}}
+
+Run the following command:
+
+```bash {location="Terminal"}
+environment:activate --resources-init={{< variable "INITIALIZATION_STRATEGY" >}}
+```
+
+For example, to use the `minimum` resource initialization strategy, run the following command:
+
+```bash {location="Terminal"}
+environment:activate --resources-init=minimum
+```
 
-### Environment sync
+### Backup restoration
+
+When you [restore a backup](/environments/restore.md),
+{{% vendor/name %}} uses the same resource allocation as the parent environment.
+If there is no parent environment, the [`default` resource initialization strategy](#default-resources) applies.
+
+You can also set another resource initialization strategy.
+
+{{% note theme="info" title="Prerequisite" %}}
+
+Make sure `sizing_api_enabled` is set to `true` in your `{{% vendor/configfile "app" %}}` file.
+
+{{% /note %}}
+
+{{< codetabs >}}
+
++++
+title=Using the CLI
++++
+
+Run the following command:
+
+```bash {location="Terminal"}
+backup:restore --resources-init={{< variable "INITIALIZATION_STRATEGY" >}}
+```
+
+For example, to use the `manual` resource initialization strategy, run the following command:
+
+```bash {location="Terminal"}
+backup:restore --resources-init=manual
+```
+
+<--->
+
++++
+title=In the Console
++++
+
+1. Navigate to the environment where you want to restore a backup.
+2. Click **Backups**.
+3. Select the backup you want to restore and click **More** {{< icon more >}} next to it.
+4. Click **Restore**.
+5. Read through the consequences.
+6. Select an initialization strategy from the proposed list.
+7. Click **Backup**.
+
+{{< /codetabs >}}
+
+## Environment sync
 
 When you [sync an environment](/glossary.md#sync),
-the target environment's disk size may be smaller that the source environment's disk size.
+changes in Production are copied to your Staging environment.
+You can synchronize only the code, only the data (databases, files), or both.
+
+As your Staging environment's disk size may be smaller than your Production environment's disk size,
+you can also sync resources.
+This ensures that there is enough disk on the Staging environment to successfully complete the sync.
```

Comment:
@nicogommen I'm unsure if I should talk about Production vs. Staging, or use more generic temrinology => source environment vs. target environment. WDYT?

Reply:
Yes, I think a generic terminology would be more appropriate here.
We could use parent and child (it synchronizes from the parent to the child env).

And it is actually not exactly working like described here.

You can synchronize only the code, only the data, or only the resources, or any combination or these.
The resources sync will actually synchronize all the resources from the parent to the child (CPU&RAM, instances and disk).

Then, if you sync the data, even if you didn't choose to sync the resources, we will do what we described before:

> In this case, to ensure that the synchronization succeeds, Upsun automatically adjusts the target environment’s disk size so it matches the source environment’s disk size.

Let me know if it's not clear 🙂